dojo/forms.py

```diff
@@ -487,8 +487,8 @@ def clean(self):
         cleaned_data = super().clean()
         scan_type = cleaned_data.get("scan_type")
         file = cleaned_data.get("file")
-        if requires_file(scan_type) and not file:
-            raise forms.ValidationError('Uploading a Report File is required for {}'.format(scan_type))
+        if file is None:
```

Comment:
There seems to be some kind of exception for SONARQUBE API import scans which don't seem to require a file?